tests/tests_pytorch/callbacks/test_pruning.py

```diff
@@ -67,6 +67,7 @@ def train_with_pruning_callback(
     accelerator="cpu",
     devices=1,
 ):
+    seed_everything(1)
     model = TestModel()
 
     # Weights are random. None is 0
```

Comment:
in the line below, very rarely can we get a weight that is equal to 0. Fixing the seed should prevent that.